assets/src/blocks/Socialshare/Socialshare.js

```diff
@@ -0,0 +1,217 @@
+import {Component,Fragment} from '@wordpress/element';
```

Comment:
Same here, this has a few indentation problems.

./node_modules/eslint/bin/eslint.js assets/src/blocks/Socialshare/Socialshare.js